src/window-service.js

```diff
@@ -242,15 +243,23 @@
 
                 this.windowTracker.add(newWindow);
 
+                var self = this;
+                var showFunction = function() {
+                    self.$timeout(() => {
+                        newWindow.show();
+                        newWindow.bringToFront();
+                    });
```

Comment:
Normally `$timeout`s are used to make angular update. What does it do here, does it break when it's removed?